scripts/relay-compiler.js

```diff
@@ -21,7 +21,7 @@ fs.writeFileSync(path.join(__dirname, '..', 'schema.graphql'), clientSchema);
 
 _x('yarn', [
   'relay-compiler',
-  '--src=./apps',
+  '--src=./',
```

Comment:
Why this change?

Reply:
Relic from times when fragment container was in `margarita-components`. 

Fixed :-)